tools/setup/postgres-init-dev-db

```diff
@@ -7,7 +7,7 @@ if [ "$(uname)" = "OpenBSD" ]; then
   DEFAULT_USER="_postgresql"
 fi
 
-ROOT_POSTGRES="sudo -u "$DEFAULT_USER" psql"
+ROOT_POSTGRES="sudo -i -u "$DEFAULT_USER" psql"
```

Comment:
Initially I raised an error here when zulip directory is not world readable that failed travis builds and on further investigation, I realised the permission error was occuring soley because my postgres user did not have access to my home dir, just making zulip dir world readable would not solve the problem. The -i flag gives the root user's environment enabling access to /home dir. To test and reproduce the error I removed packages and then tried the two commands, `sudo -u postgres psql` which gave the same cd error and `sudo -i -u postgres psql` which worked, also in a virtualbox setup of Ubuntu the same worked.